keps/sig-apps/3998-job-success-completion-policy/kep.yaml

```diff
@@ -5,7 +5,7 @@ authors:
 owning-sig: sig-apps
```

Comment:
I'm missing the following updates in the README.md part of this enhancement:
1. Check appropriate boxes in the Release Signoff Checklist
2. You'll want to update integration and e2e tests sections with links to triage board, see [template](https://github.com/kubernetes/enhancements/blob/master/keps/NNNN-kep-template/README.md#integration-tests).
3. One of the GA criteria [mentions](https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/3998-job-success-completion-policy#ga) mentions verifying `Complete` condition, what are the plans? I'd expect at this point in time this wording to be change do we will or we won't. 
4. Update Implementation history with this edit information.

Reply:
Let me check those, then share them with you here.

Reply:
@soltysh I addressed them in [9b465bb](https://github.com/kubernetes/enhancements/pull/5148/commits/9b465bbd2892643293f4a5afa74073bcd0a1ba23).

Regarding 4, it would be better to "will do" in this GA graduation since we introduced the dedicated `Completions` reason, `CompletionsReached` for exiting Completions behavior and `SuccessPolicy` for new Completions with SuccessPolicy behavior.